packages/rexray/extra/dcos-rexray.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=REX-Ray: A vendor agnostic storage orchestration engine
+
+[Service]
+StartLimitInterval=0
+RestartSec=15
+Restart=always
+EnvironmentFile=/etc/environment
```

Comment:
`/etc/environment` doesn't exist many places. `/opt/mesosphere/environment` should be plenty.